events/class_tests/participant_test.py

```diff
@@ -48,3 +49,25 @@ def test_exist_event_participant(self):
     def test_invalid_register_user_twice(self, persist_event_participant):
         with pytest.raises(ValidationError, match='user already exist in meeting'):
             persist_event_participant.save()
+
+    def test_get_an_event_participants(self):
+        event = Event.objects.get(title="event1")
+        expected_participants = [
+            EventParticipant.objects.get(id=1),
+            EventParticipant.objects.get(id=2)
+        ]
+        all_participants_in_event1 = EventParticipant.objects.get_an_event_participants(event)
+        assert list(expected_participants) == list(all_participants_in_event1)
```

Comment:
expected_participants defined as list in line # 55 so I think you can do it like:
```suggestion
        assert expected_participants == list(all_participants_in_event1)
```

Reply:
Done 😄

Reply:
Done 😄